python/ray/tests/test_cancel.py

```diff
@@ -23,7 +23,7 @@ def valid_exceptions(use_force):
 def test_cancel_chain(ray_start_regular, use_force):
     signaler = SignalActor.remote()
 
-    @ray.remote
+    @ray.remote(num_cpus=0)
```

Comment:
These changes are necessary because the test relies on running more than num_cpus of `wait_for` concurrently.